pkg/ingester/user_state.go

```diff
@@ -168,7 +169,7 @@ func (u *userState) getSeries(metric model.Metric, cfg *UserStatesConfig) (model
 		return fp, nil, httpgrpc.Errorf(http.StatusTooManyRequests, "per-user series limit (%d) exceeded", cfg.MaxSeriesPerUser)
 	}
 
-	metricName, err := util.ExtractMetricNameFromMetric(metric)
+	metricName, err := extract.MetricNameFromMetric(metric)
```

Comment:
Why isn't this change part of the previous commit?

Reply:
Oh probably got mixed up in various rebases; should be fixed now.